User documentation for the PlotSelection feature (#4146)
silx-kit/silx: doc/source/applications/view.rst

   * - |imgViewHdf5|
     - Viewing metadata and HDF5 attributes

Custom plot selection
---------------------

.. grid :: 1 2 2 2

   .. grid-item-card::
      :columns: 12
      :text-align: center

      .. figure:: img/silx-view-plotSelection.png
         :align: center

>       Functionality enabling you to choose the dataset you want for the x-axis and y-axis and display several curves at once.

Reviewer: Let's make it short:
```suggestion
      Plot custom curves from selected 1D datasets.
```

How to access :
............

.. grid :: 1 2 2 2

   .. grid-item-card::
      :columns: 12
      :text-align: center

      .. figure:: img/silx-view-plotMenu.png
         :align: center

>       Open the 'plot selection' window (View -> plot selection)

Reviewer: Let's keep upper case consistent with the name of the menu and window
```suggestion
      Open the 'Plot selection' window (View menu -> Plot selection)
```

      .. figure:: img/silx-view-dropPlot.png
         :align: center

>       The new window opens and you can drag and drop the dataset you want to plot or to x-axis and y-axis.

Reviewer: ```suggestion
      The "Plot selection" window opens: You can plot custom curves by dragging and dropping 1D datasets from "silx view" to "Plot selection" windows "X" and "Y" fields.
```

  --hdf5-file-locking   Start the application with HDF5 file locking enabled (it is disabled by
                        default)

> Examples
> -----------------

Reviewer: The underline should have the same length as the title:
```suggestion
Examples
--------
```
